encoding/ssh/filexfer/fx.go

```diff
@@ -0,0 +1,137 @@
+package filexfer
+
+import (
+	"fmt"
+)
+
+// Status defines the SFTP error codes used in SSH_FXP_STATUS response packets.
+type Status uint32
+
+// Defines the various SSH_FX_* values.
+const (
+	// see draft-ietf-secsh-filexfer-02
+	// https://tools.ietf.org/html/draft-ietf-secsh-filexfer-02#section-7
+	StatusOK = Status(iota)
```

Comment:
godoc doesn't recognize this form. It does recognize

```go
StatusOK Status = iota
```

and puts the constants below the type definition. (Not sure what pkg.go.dev does, but it might be the same.)